README.md

```diff
@@ -39,11 +39,24 @@ Fraunhofer FOKUS has developed the Open Content Decryption Module (OCDM) accordi
 * build the following target (check *[Build notes for Chromium](docs/build_notes_chromium.md)* section before building)  
   ```# ninja -C out/Debug chrome opencdmadapter```
 
+### ...for WPE Browser
+* add meta-metrological layer to yocto layer's directory   
+  ```# git clone https://github.com/Metrological/meta-metrological```   
+  ```# bitbake-layers add-layer ../meta-metrological``` absolute or reference path of meta-metrological layer to be mentioned   
+* ensure 'opencdm' is added to the PACKAGECONFIG of wpewebkit recipe   
+* build WPE   
+  ```# bitbake wpewebkit```   
+  Note: Linaro [CDM](https://github.com/linaro-home/open-content-decryption-module) to be built before building WPE webkit and it's recipe yet to be uploaded into meta-linaro yocto layer   
```

Comment:
Regarding the OE integration of OpenCDM for WPE browser...

 I think the correct thing to do with how it is currently implemented would be to add a dependency on the OpenCDM recipe to the https://github.com/linaro-home/meta-lhg/blob/morty/meta-lhg-wpe/recipes-wpe/wpewebkit/wpewebkit_0.1.bbappend. Also a separate pull request needs to be sent to add this recipe to upstream (I think to the meta-lhg layer).

However on Chromium opencdm didn't have its own recipe, it was integrated as a bbapend to the chromium recipe (see here https://github.com/linaro-home/meta-lhg/blob/morty/meta-lhg/recipes-browser/chromium/chromium-wayland_%25.bbappend).

It doesn't seem to make to much sense integrating it differently for another browser.

@andrey-konovalov do you have any thoughts on what is the best way to integrate ocdm (as a bbapend, or a separate recipe? 

@moorthy-bs was there any reason to do this as a separate recipe?

Reply:
@petegriffin Metrological layer have opencdm https://github.com/WebPlatformForEmbedded/meta-wpe/blob/morty/recipes-drm/opencdm/opencdm_git.bb as separate recipe and linking it(libocdm.so) only when opencdm  packageconfig is enabled.

Reply:
For chromium ocdm is built as pepper plugin, and this is done as part of the chromium build. Hence the bbappend.
As far as I can see, wpewebkit uses different build system, and probably doesn't support Pepper Plugin API at all. Then building and running ocdm for wpewebkit the same way like it was done for chromium is not possible.

Reply:
I don't quite understand the following test in README.md  "Note: Linaro [CDM](https://github.com/linaro-home/open-content-decryption-module) to be built before building WPE webkit and it's recipe yet to be uploaded into meta-linaro yocto layer "

The recipe appears already to be merged to WPE layer, so we should be using that right?

It looks like the correct way to integrate with OE will be to have a bbappend to override SRC_URI in meta-wpe/blob/morty/recipes-drm/opencdm/opencdm_git.bb to point at linaro-home OpenCDM repo.

Then also another .bbapend in meta-lhg to add add OpenCDM to the packageconfig of WPE if we are building for an OPTEE enabled board (see https://github.com/linaro-home/meta-lhg/blob/morty/meta-lhg/recipes-browser/chromium/chromium-wayland_%25.bbappend#L1 as an example).

Reply:
Okay, let me verify SRC_URI override then go for pull request in meta-lhg